src/ServiceControl.Monitoring/Http/Diagrams/Aggregator.cs

```diff
@@ -22,27 +22,22 @@ internal static MonitoredValues AggregateTimings<T>(List<IntervalsStore<T>.Inter
             };
         }
 
-        internal static MonitoredValues AggregateQueueLength<T>(List<IntervalsStore<T>.IntervalsBreakdown> intervals, HistoryPeriod period)
+        internal static MonitoredValues ToSumOfBreakdownAverages<T>(List<IntervalsStore<T>.IntervalsBreakdown> intervals, HistoryPeriod period)
         {
             Func<long, double> returnOneIfZero = x => x == 0 ? 1 : x;
 
             return new MonitoredValues
             {
-                // To calculate the average we count TotalMeasurements instead of the number of intervals. 
```

Comment:
@Scooletz this comment was removed as previously this piece of code was QL specific (method was called `AggreateQueueLength` instead of `ToSumOfBreakdownAverages`). I assumed that method rename will make it more clear what to expect.

Reply:
I retract my comment about removing the comment. Now, after looking at the implementation and the name of the method, it's a well defined function (a sum of averages) which is the last but one line in the LINQ statement